test/lib/tasks/receptor/extract_orgs_test.rb

```diff
@@ -39,7 +39,7 @@ def test_output_file
 
       Rake.application.invoke_task('katello:receptor:extract_orgs')
 
-      data = JSON.parse(File.read(File.open(ENV['OUTPUT_FILE'])))
+      data = JSON.parse(File.read(ENV['OUTPUT_FILE']))
```

Comment:
I think this is the same, but please check. Might also have been a bug.

Reply:
The original was unnecessary syntax in 2.5, but not functionally a defect.  Good catch, thank you.